src/main/java/com/stories/service/StoriesServiceImpl.java

```diff
@@ -126,6 +128,64 @@ public StoryDomain updateStory(StoryDomain storyDomain, String id) throws Except
 			throw new EntityNotFoundException(mensaggeDinamicValidation[0], mensaggeDinamicValidation[1]);
 		}
 	}
+	
```

Comment:
Please, verify any unnecessary blank space between the code lines and any unnecessary imports on this class please.

Reply:
Solved